Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 RAILS_VERSION = '~> 4.1.8'
 
-ruby ENV['GEMFILE_RUBY_VERSION'] if ENV['GEMFILE_RUBY_VERSION']
+send :ruby, ENV['GEMFILE_RUBY_VERSION'] if ENV['GEMFILE_RUBY_VERSION']
```

Comment:
We could change it to:
```ruby
if ENV['GEMFILE_RUBY_VERSION']; ruby ENV['GEMFILE_RUBY_VERSION']; end
```
if you like that better. Putting it all on one line seems to defeat rvm's regex.